astropy/coordinates/earth.py

```diff
@@ -655,21 +655,26 @@ def to_geocentric(self):
         """Convert to a tuple with X, Y, and Z as quantities."""
         return (self.x, self.y, self.z)
 
-    def get_itrs(self, obstime=None):
+    def get_itrs(self, obstime=None, location=None):
         """
         Generates an `~astropy.coordinates.ITRS` object with the location of
-        this object at the requested ``obstime``.
+        this object at the requested ``obstime``, relative to the given
+        ``location`` (if provided).
 
         Parameters
         ----------
         obstime : `~astropy.time.Time` or None
-            The ``obstime`` to apply to the new `~astropy.coordinates.ITRS`, or
-            if None, the default ``obstime`` will be used.
+            This object at the requested ``obstime``, either geocentric, or
```

Comment:
Actually, I now see that you applied my suggested at the wrong place. The description for `obstime` should be kept as is!